book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book)
+    @rentals.push(book)
+  end
```

Comment:
- [ ] A rental needs to have a book and a person, along with a date. I think we can mirror what we have in the Person class and change this method to this:

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```
  This way the entire rental object is added to the `rentals` array.